adsft/extraction.py

```diff
@@ -811,6 +810,7 @@ def extract_multi_content(self, translate=False, decode=True):
                     grobid_xml = response.text
                 else:
                     logger.error("Grobid service response error (code %s): %s", response.status_code, response.text)
+            ft_source.close()

```

Comment:
the trouble here is that ft_source won't exist; so this will cause another exception